featuretools/tests/computational_backend/test_calculate_feature_matrix.py

```diff
@@ -1743,7 +1743,7 @@ def error(s):
         calculate_feature_matrix([value, error_feature],
                                  es,
                                  verbose=True)
-        assert False
+        assert False  # pragma: no cover
```

Comment:
can this line reasonably be hit?

Reply:
Are you suggesting we take out the line completely?

Reply:
It could be that this assertion was placed here to indicate that whatever comes before it is intended to cover all cases.  I sometimes do this, for example, if I have a string of if statements that are meant to cover all the possible types of values that might be passed to a function.  Then, I'll include a final `else:` that just has something like `raise Exception("invariant")` or `assert False`.  It shows very clearly that the if statements were supposed to handle any kind of value that came in and that they should have explicitly listed out the supported value types.

Reply:
Fixed with pytest.raises:
https://github.com/alteryx/featuretools/pull/1829/commits/82baa04e986d6dce85847ed3b4b097a9911c5f91